### DISCUSSION TITLE
Feature: Footer

### DISCUSSION THREAD

**Discussion by FlyteWizard** [VikeLabs/courseup#210] [Ideas]
To stop the header from being too crowded, and to convey more information to users, it might be nice to have a footer.

This will help with the overlap issue for the component in the bottom right corner, by moving the information to the footer, and will allow for the header/navigation to remain uncluttered.

**Comment by keithradford** [comment]
Do you have an example of a website with something similar? I'm all for de-cluttering but not sure about having a header *and* footer. Maybe is more beneficial for mobile?

**Reply by FlyteWizard** [reply]
I'm just basing this off the feature request: https://github.com/VikeLabs/courseup/pull/195

And the fact that cluttered headers/navigations are overwhelming, and hard to navigate with keyboard if you want to get to something easy, and not everything belongs in the header, some things are better for footer (GitHub link, feedback link, about, status, legal, etc.).

**Reply by aomi** [reply]
The header is pretty much at it's limit unless we start "collapsing" things that's for sure. I can see a _thin_ footer being not a bad idea. I can't find any examples of a footer where the app primarily consists of scroll-able elements (ie. this app and Discord). Most sites with a footer have either a sticky header and the rest of the page is scroll-able with a footer.

**Reply by FlyteWizard** [reply]
I like the idea of a thin footer, it doesn't need to be fancy.

This is what GitHub uses...
<img width="1280" alt="Capture d’écran, le 2021-06-13 à 13 26 55" src="https://user-images.githubusercontent.com/10472448/121821030-0ac18400-cc4b-11eb-8fd3-d3202889d45e.png">

Maybe something similar?

* Contribute / GitHub
* Feedback
* VikeLabs
